Makefile

```diff
@@ -126,6 +126,17 @@ sepolia: compile-all
 test: compile-all
 	mix test --no-start --exclude spectest
 
+#### BEACON NODE OAPI ####
+OAPI_NAME = beacon-node-oapi
+OAPI_VERSION := $(shell cat .oapi_version)
+$(OAPI_NAME).json:
+	curl -L -o "$@" \
+		"https://ethereum.github.io/beacon-APIs/releases/${OAPI_VERSION}/beacon-node-oapi.json"
+
+OPENAPI_JSON := $(OAPI_NAME).json 
```

Comment:
maybe we should add this as a dependency to `compile-all`. @MegaRedHand what do you think?

Reply:
Yes, probably. The project wouldn't compile without it. Maybe we'll also have to update the CI 🤔

Reply:
yes, it is necessary to update the CI :sweat_smile: otherwise the build process fails.
I am going to try to update that part